### PR TITLE
[zh] sync v1.24 concepts-6

### PR DESCRIPTION
**Pull request by JarHMJ**
related issues: #33476 concepts-6

### CODE REVIEW

File: content/zh/docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
...
+A node shutdown action may not be detected by kubelet's Node Shutdown Mananger, 
+either because the command does not trigger the inhibitor locks mechanism used by 
+kubelet or because of a user error, i.e., the ShutdownGracePeriod and 
+ShutdownGracePeriodCriticalPods are not configured properly. Please refer to above 
+section [Graceful Node Shutdown](#graceful-node-shutdown) for more details.
+-->
+节点关闭的操作可能无法被 kubelet 的节点关闭管理器检测到，
+是因为该命令不会触发 kubelet 所使用的抑制锁定机制，或者是因为用户错误的原因，
+即 ShutdownGracePeriod 和 ShutdownGracePeriodCriticalPod 配置不正确。
+请参考以上[节点体面关闭](#graceful-node-shutdown) 部分了解更多详细信息。
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/nodes.md:852]:
```suggestion
请参考以上[节点体面关闭](#graceful-node-shutdown)部分了解更多详细信息。
```

File: content/zh/docs/concepts/architecture/nodes.md [hunk 2]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
...
+the shutdown node is not available to delete the pods so the StatefulSet cannot 
+create a new pod with the same name. If there are volumes used by the pods, the 
+VolumeAttachments will not be deleted from the original shutdown node so the volumes 
+used by these pods cannot be attached to a new running node. As a result, the 
+application running on the StatefulSet cannot function properly. If the original 
+shutdown node comes up, the pods will be deleted by kubelet and new pods will be 
+created on a different running node. If the original shutdown node does not come up,  
+these pods will be stuck in terminating status on the shutdown node forever.
+-->
+当节点关闭但 kubelet 的节点关闭管理器未检测到的时候，
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/nodes.md:867]:
```suggestion
当某节点关闭但 kubelet 的节点关闭管理器未检测到这一事件时，
```

File: content/zh/docs/concepts/architecture/nodes.md [hunk 3]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
...
+在那个关闭节点上的属于 StatefulSet 的 Pod 将变成终止状态，并且不能移动到新的运行节点上。
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/nodes.md:868]:
```suggestion
在那个已关闭节点上、属于 StatefulSet 的 Pod 将停滞于终止状态，并且不能移动到新的运行节点上。
```

File: content/zh/docs/concepts/architecture/nodes.md [hunk 4]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
...
+这是因为 kubelet 在关闭节点上无法删除 pod，
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/nodes.md:869]:
```suggestion
这是因为已关闭节点上的 kubelet 已不存在，亦无法删除 Pod，
```

File: content/zh/docs/concepts/architecture/nodes.md [hunk 5]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
...
+因此 StatefulSet 无法创建一个具有相同名称的新 pod。
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/nodes.md:870]:
```suggestion
因此 StatefulSet 无法创建同名的新 Pod。
```

File: content/zh/docs/concepts/architecture/nodes.md [hunk 6]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
...
+如果有 Pod 使用的卷，则 VolumeAttachments 不会从原始关闭节点中删除，
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/nodes.md:871]:
```suggestion
如果 Pod 使用了卷，则 VolumeAttachments 不会从原来的已关闭节点上删除，
```

File: content/zh/docs/concepts/architecture/nodes.md [hunk 7]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
...
+因此这些 pod 使用的卷不能附加到新的运行节点。
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/nodes.md:872]:
```suggestion
因此这些 Pod 所使用的卷也无法挂接到新的运行节点上。
```

File: content/zh/docs/concepts/architecture/nodes.md [hunk 8]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
...
+所以，那些以 StatefulSet 运行的应用程序无法正常运行。
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/nodes.md:873]:
```suggestion
所以，那些以 StatefulSet 形式运行的应用无法正常工作。
```

File: content/zh/docs/concepts/architecture/nodes.md [hunk 9]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
...
+如果原来关闭节点恢复后，kubelet 将删除 pods，新的 pods 将被在不同的运行节点上创建。
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/nodes.md:874]:
```suggestion
如果原来的已关闭节点被恢复，kubelet 将删除 Pod，新的 Pod 将被在不同的运行节点上创建。
```

File: content/zh/docs/concepts/architecture/nodes.md [hunk 10]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
...
+如果原来关闭节点没有恢复，那些在关闭节点上 pod 将永远停留在终止状态。
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/nodes.md:875]:
```suggestion
如果原来的已关闭节点没有被恢复，那些在已关闭节点上的 Pod 将永远滞留在终止状态。
```

File: content/zh/docs/concepts/architecture/nodes.md [hunk 11]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
...
+If the `NodeOutOfServiceVolumeDetach`  [feature gate](/docs/reference/
+command-line-tools-reference/feature-gates/) is enabled on
+`kube-controller-manager`, and a Node is marked out-of-service with this taint, the 
+pods on the node will be forcefully deleted if there are no matching tolerations on
+it and volume detach operations for the pods terminating on the node will happen
+immediately. This allows the Pods on the out-of-service node to recover quickly on a
+different node. 
+-->
+为了缓解上述情况，用户可以手动将具有 `NoExecute` 或 `NoSchedule` 效果的
+`node kubernetes.io/out-of-service` 污点添加到标记为停止服务的节点上。
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/nodes.md:890]:
```suggestion
`node kubernetes.io/out-of-service` 污点添加到节点上，标记其无法提供服务。
```

File: content/zh/docs/concepts/architecture/nodes.md [hunk 12]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
...
+如果在 `kube-controller-manager` 上启用了 `NodeOutOfServiceVolumeDetach` 
+[特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)，
+并且节点被标记为停止服务这种污点，如果节点上没有匹配的容忍度，
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/nodes.md:893]:
```suggestion
并且节点被通过污点标记为无法提供服务，如果节点 Pod 上没有设置对应的容忍度，
```

File: content/zh/docs/concepts/architecture/nodes.md [hunk 13]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
...
+那节点上的 pod 将被强制删除，并且该在节点上被终止的 pod 将立即进行卷分离操作。
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/nodes.md:894]:
```suggestion
那么这样的 Pod 将被强制删除，并且该在节点上被终止的 Pod 将立即进行卷分离操作。
```

File: content/zh/docs/concepts/architecture/nodes.md [hunk 14]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
...
+这允许那些在停止服务节点上的 Pod 能在其他节点上快速恢复。
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/nodes.md:895]:
```suggestion
这样就允许那些在无法提供服务节点上的 Pod 能在其他节点上快速恢复。
```

File: content/zh/docs/concepts/architecture/nodes.md [hunk 15]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
...
+
+<!--
+During a non-graceful shutdown, Pods are terminated in the two phases:
+
+1. Force delete the Pods that do not have matching `out-of-service` tolerations.
+2. Immediately perform detach volume operation for such pods. 
+-->
+在非体面关闭期间，Pod 分两个阶段终止：
+1. 强制删除没有匹配的 `out-of-service` 容忍度的 Pod。
+2. 立即对此类 pod 执行分离卷操作。
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/nodes.md:905]:
```suggestion
2. 立即对此类 Pod 执行分离卷操作。
```

**Comment by tengqm** [comment]
@JarHMJ Please submit PR for the two files in 'concepts-6'.

**Comment by JarHMJ** [comment]
> @JarHMJ Please submit PR for the two files in 'concepts-6'.
> 
> You can check how #33561 is organized.

I have synced the two files.

File: content/zh/docs/concepts/architecture/nodes.md [hunk 16]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
 ```
 {{< /note >}}
 
+<!--
+## Non Graceful node shutdown {#non-graceful-node-shutdown}
+-->
+## 节点非体面关闭 {#non-graceful-node-shutdown}
```
Comment (xichengliudui) [inline content/zh/docs/concepts/architecture/nodes.md:838]: 体面 -> 优雅
大家都喜欢叫做：“优雅退出”

File: content/zh/docs/concepts/architecture/nodes.md [hunk 17]
```diff
@@ -832,6 +832,94 @@ Message:        Pod was terminated in response to imminent node shutdown.
...
+A node shutdown action may not be detected by kubelet's Node Shutdown Mananger, 
+either because the command does not trigger the inhibitor locks mechanism used by 
+kubelet or because of a user error, i.e., the ShutdownGracePeriod and 
+ShutdownGracePeriodCriticalPods are not configured properly. Please refer to above 
+section [Graceful Node Shutdown](#graceful-node-shutdown) for more details.
+-->
+节点关闭的操作可能无法被 kubelet 的节点关闭管理器检测到，
+是因为该命令不会触发 kubelet 所使用的抑制锁定机制，或者是因为用户错误的原因，
+即 ShutdownGracePeriod 和 ShutdownGracePeriodCriticalPod 配置不正确。
+请参考以上[节点体面关闭](#graceful-node-shutdown)部分了解更多详细信息。
```
Comment (xichengliudui) [inline content/zh/docs/concepts/architecture/nodes.md:852]: 同上

File: content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 18]
```diff
@@ -12,37 +12,40 @@ weight: 10
...
 
 * `cni-bin-dir`: Kubelet probes this directory for plugins on startup
 * `network-plugin`: The network plugin to use from `cni-bin-dir`.  It must match the name reported by a plugin probed from the plugin directory.  For CNI plugins, this is "cni".
 -->
 ## 安装
 
-kubelet 有一个单独的默认网络插件，以及一个对整个集群通用的默认网络。
-它在启动时探测插件，记住找到的内容，并在 Pod 生命周期的适当时间执行
-所选插件（这仅适用于 Docker，因为 CRI 管理自己的 CNI 插件）。
+CNI 插件需要实现[Kubernetes 网络模型](/zh/docs/concepts/services-networking/#the-kubernetes-network-model)。
```
Comment (xichengliudui) [inline content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:47]:
```suggestion
CNI 插件需要实现 [Kubernetes 网络模型](/zh/docs/concepts/services-networking/#the-kubernetes-network-model)。
```

File: content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 19]
```diff
@@ -12,37 +12,40 @@ weight: 10
...
+在更广泛的 Kubernetes 生态系统中你可以使用不同的插件（开源和闭源）。
+
+<!-- 
+You must use a CNI plugin that is compatible with the 
+[v0.4.0](https://github.com/containernetworking/cni/blob/spec-v0.4.0/SPEC.md) or later
+releases of the CNI specification. The Kubernetes project recommends using a plugin that is
+compatible with the [v1.0.0](https://github.com/containernetworking/cni/blob/spec-v1.0.0/SPEC.md)
+CNI specification (plugins can be compatible with multiple spec versions). 
+-->
+你必须使用与[v0.4.0](https://github.com/containernetworking/cni/blob/spec-v0.4.0/SPEC.md)
```
Comment (xichengliudui) [inline content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:30]:
```suggestion
你必须使用与[v0.4.0](https://github.com/containernetworking/cni/blob/spec-v0.4.0/SPEC.md)或更高版本兼容的 CNI 插件。
```

File: content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 20]
```diff
@@ -12,37 +12,40 @@ weight: 10
...
+或更高版本的 CNI 规范相符合的 CNI 插件。
```
Comment (xichengliudui) [inline content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:31]: 删除。

File: content/zh/docs/concepts/architecture/nodes.md [hunk 21]
```diff
@@ -961,17 +1049,33 @@ kubelet 会直接跳到下一个优先级数值范围进行处理。
...
 
+<!-- 
+{{< note >}}
+The ability to take Pod priority into account during graceful node shutdown was introduced
+as an Alpha feature in Kubernetes v1.23. In Kubernetes {{< skew currentVersion >}}
+the feature is Beta and is enabled by default.
+{{< /note >}} 
+-->
+{{< note >}}
+在体面节点关闭期间考虑 Pod 优先级的能力是作为 Kubernetes v1.23 中的 Alpha 功能引入的。
```
Comment (xichengliudui) [inline content/zh/docs/concepts/architecture/nodes.md:1075]: 同上

File: content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 22]
```diff
@@ -12,37 +12,39 @@ weight: 10
...
+在更广泛的 Kubernetes 生态系统中你可以使用不同的插件（开源和闭源）。
+
+<!-- 
+You must use a CNI plugin that is compatible with the 
+[v0.4.0](https://github.com/containernetworking/cni/blob/spec-v0.4.0/SPEC.md) or later
+releases of the CNI specification. The Kubernetes project recommends using a plugin that is
+compatible with the [v1.0.0](https://github.com/containernetworking/cni/blob/spec-v1.0.0/SPEC.md)
+CNI specification (plugins can be compatible with multiple spec versions). 
+-->
+你必须使用与[v0.4.0](https://github.com/containernetworking/cni/blob/spec-v0.4.0/SPEC.md)或更高版本的 CNI 规范相符合的 CNI 插件。
```
Comment (tengqm) [inline content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:30]:
```suggestion
你必须使用与 [v0.4.0](https://github.com/containernetworking/cni/blob/spec-v0.4.0/SPEC.md)
或更高版本的 CNI 规范相兼容的 CNI 插件。
```

File: content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 23]
```diff
@@ -12,37 +12,39 @@ weight: 10
...
+Kubernetes 推荐使用一个兼容[v1.0.0](https://github.com/containernetworking/cni/blob/spec-v1.0.0/SPEC.md)
```
Comment (tengqm) [inline content/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:31]:
```suggestion
Kubernetes 推荐使用一个兼容 [v1.0.0](https://github.com/containernetworking/cni/blob/spec-v1.0.0/SPEC.md)
```

**Comment by JarHMJ** [comment]
@xichengliudui need lgtm

**Pull request merged**